far/FarEng.hlf.m4

```diff
@@ -3434,6 +3434,12 @@ string is changed to that of the found string. For example:
 $ #Editor: All matching entries menu#
  The following key combinations are available in this menu:
 
+ #F2#
+ Copy all found lines into a new editor and close this menu.
+
+ #Shift+F2#
+ Copy all lines matching the current filter into a new editor and close this menu.
```

Comment:
Unfortunately, it's not clear what `the current filter` is and how this action is different from **`F2`**.